jsoc/projects.md

```diff
@@ -42,4 +42,5 @@ We also have Julia project's available under other organizations. If you are app
 * [SciML - NumFocus](https://github.com/numfocus/gsoc#sciml)
   - [SciML](/jsoc/gsoc/sciml/) - Scientific Machine Learning
   - [Differential Equations](/jsoc/gsoc/diffeq/) - Numerical methods for high-performance solving of differential equation models.
+* FluxML - NumFocus
```

Comment:
It seems like we will need to also make a PR to https://github.com/numfocus (is this right @logankilpatrick ?). Can we point to `/jsoc/gsoc/flux` as our "ideas page"?